### PR TITLE
XW-3111 | Prevent to show featured video on history/delete/protect pages

### PR DESCRIPTION
**Pull request by rybmat**
https://wikia-inc.atlassian.net/browse/XW-3111

@Wikia/x-wing

### CODE REVIEW

**Review by latata — APPROVED** [review]

File: extensions/wikia/ArticleVideo/ArticleVideoContext.class.php [hunk 1]
```diff
@@ -56,4 +57,9 @@ public static function getRelatedVideoData( $title ) {
 	public static function isRelatedVideoEmbedded( $title ) {
 		return !empty( static::getRelatedVideoData( $title ) );
 	}
+
+
+	private static function isArticlePage() {
```
Comment (kvas-damian) [inline extensions/wikia/ArticleVideo/ArticleVideoContext.class.php]: maybe `isArticleView`?
Should it be a part of class that can be used in other places? Maybe `Tititle` or something?

Comment (rybmat) [reply extensions/wikia/ArticleVideo/ArticleVideoContext.class.php]: @kvas-damian what about WikiaPageType class?

Comment (kvas-damian) [reply extensions/wikia/ArticleVideo/ArticleVideoContext.class.php]: works for me

Comment (rybmat) [reply extensions/wikia/ArticleVideo/ArticleVideoContext.class.php]: even better: https://github.com/Wikia/app/blob/efb59c78e5eb20a32473581dd4643e3273eceb33/includes/wikia/WikiaPageType.class.php#L219-L219
I missed it last time I was looking there

**Review by latata — APPROVED** [review]

**Review by kvas-damian — APPROVED** [review]

**Pull request merged**